### DISCUSSION TITLE
How are the images generated?

### DISCUSSION THREAD

**Discussion by dwightmulcahy** [BitMaker-hub/NerdMiner_v2#354] [General]
How are the images generated for Nerdminer?  I see that they are stored in unsigned short array in the NerdMiner_v2/tree/master/src/media directory but no comments as to *how* they are generated.

**Comment by BitMaker-hub** [comment]
Hi, sorry for a late answer on this.

You can use literally this link to switch from an image to its array in C: https://notisrac.github.io/FileToCArray/
The array should be in 565 image format. 
I also used an .exe solution that let you do this too: ImageConverter565, it makes exactly the same

I'll post the original images on the next release, and a way to create your customs screens explanation
